### DISCUSSION TITLE
[Question] Keyboard combination

### DISCUSSION THREAD

**Discussion by ingdesarr1** [flet-dev/flet#3542] [Q&A] [answered]
### Question

Good afternoon. How can I read a key combination? for example, read ctrl+alt+u, to execute an action within flet? Thanks

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by ndonkoHenri** [comment, marked as answer]
https://flet.dev/docs/cookbook/keyboard-shortcuts
